src/duplo_resource/jit.py

```diff
@@ -47,6 +47,47 @@ def token(self) -> dict:
       token: The JWT token.
     """
     return {"token": self.duplo.token}
+  
+  @Command()
+  def gcp(self, nocache: bool = None) -> dict:
+    """GCP Access Token
+    
+    Get the GCP JWT token for the current user. This is the token that is used to authenticate with the GCP API. 
+
+    Example: Using for gcloud cli access
+      Here is how to set the needed environment variables for the gcloud cli.
```

Comment:
This works only for admin user. Should we mention it?

Reply:
Yeah. I also wonder why even have the tenant id in the url?